psga/config/psga-schema-s-aureus.yaml

```diff
@@ -25,7 +25,7 @@ pipeline:
         slug: ont
       pathogen:
         name: Staphylococcus aureus
-        slug: s-aureus
+        slug: s_aureus
```

Comment:
Because we used an underscore when we set up the pipeline manually in the dev system. I can change it to a dash, but we'll have to use a dash when we set up whatever system we're delivering to HSA

Reply:
I'll change it to a -

Reply:
Fixed